client/state/ui/checkout/test/reducer.js

```diff
@@ -110,6 +110,14 @@ describe( 'reducer', () => {
 			expect( state ).toBe( 'themes' );
 		} );
 
+		test( 'should return hosting for hosting', () => {
+			const state = upgradeIntent( 'blastendedskrewt', {
```

Comment:
For folks reading the first argument is what the current upgrade intent is, usually a section name. 

Also had to look this one up, lol:
<img width="255" alt="Screen Shot 2019-12-09 at 1 55 46 PM" src="https://user-images.githubusercontent.com/1270189/70476632-f74cc000-1a8b-11ea-8f30-44bb3e3849b7.png">